test/support/conn_case.ex

```diff
@@ -36,8 +38,23 @@ defmodule CadetWeb.ConnCase do
     conn = Phoenix.ConnTest.build_conn()
 
     if tags[:authenticate] != nil do
```

Comment:
Prefer `tags[:authenticate]` to `!=nil`

Reply:
copied it from your branch :upside_down_face: